examples/private_connection/main.tf

```diff
@@ -0,0 +1,190 @@
+  description = "private zone for Google API private access"
+  visibility  = "private"
+  private_visibility_config {
+    networks {
+      network_url = google_compute_network.network.self_link
+    }
+  }
+}
+
+resource "google_dns_record_set" "forseti-private-zone-cname" {
+  managed_zone = google_dns_managed_zone.forseti-private-zone.name
+  name         = "*.${google_dns_managed_zone.forseti-private-zone.dns_name}"
+  type         = "CNAME"
+  ttl          = 300
+  rrdatas      = ["private.googleapis.com."]
+}
+
+resource "google_dns_record_set" "forseti-private-zone-a" {
+  managed_zone = google_dns_managed_zone.forseti-private-zone.name
+  name         = "private.${google_dns_managed_zone.forseti-private-zone.dns_name}"
+  type         = "A"
+  ttl          = 300
+  rrdatas      = ["199.36.153.8", "199.36.153.10", "199.36.153.9", "199.36.153.11"]
+}
+
+# Route calls to private API via default-internet-router
+resource "google_compute_route" "forseti-private-vip-route" {
+  name             = "forseti-private-vip-route"
+  dest_range       = "199.36.153.8/30"
+  priority         = 100
+  network          = google_compute_network.network.name
+  next_hop_gateway = "default-internet-gateway"
+}
+
+# Forseti deployment.
+module "forseti" {
+  source               = "../../"
+  domain               = var.domain
+  project_id           = var.project_id
+  org_id               = var.org_id
+  manage_rules_enabled = false
+  composite_root_resources = [
+    "organizations/${var.org_id}",
+  ]
+  config_validator_enabled = true
+  server_private           = true
+  client_private           = true
+  cloudsql_private         = true
+  server_region            = var.region
+  client_region            = var.region
+  cloudsql_region          = var.region
+  storage_bucket_location  = var.region
+  bucket_cai_location      = var.region
+  server_boot_image        = "gce-uefi-images/ubuntu-1804-lts"
+  client_boot_image        = "gce-uefi-images/ubuntu-1804-lts"
+  network                  = google_compute_network.network.name
+  subnetwork               = google_compute_subnetwork.subnet.name
+}
+
+resource "google_compute_firewall" "forseti-server-allow-sql-connection" {
+  name                    = "forseti-server-allow-sql-connection"
+  network                 = google_compute_network.network.name
+  direction               = "EGRESS"
+  target_service_accounts = [module.forseti.forseti-server-service-account]
+  destination_ranges      = [module.forseti.forseti-cloudsql-instance-ip]
+  priority                = "1000"
+
+  allow {
+    protocol = "tcp"
+    ports    = ["3306", "3307"]
+  }
+}
+
+resource "google_compute_firewall" "forseti-server-allow-internet" {
+  name                    = "forseti-server-allow-internet"
+  network                 = google_compute_network.network.name
+  direction               = "EGRESS"
+  target_service_accounts = [module.forseti.forseti-server-service-account]
+  destination_ranges      = ["0.0.0.0/0"]
```

Comment:
Added an additional bool input variable called `post_install` to control the destination range here.

`var.post_install ? ["199.36.153.8/30"] : ["0.0.0.0/0"]`